Accepted-RFCs/RFC-013-new-initiative-process.md

```diff
@@ -0,0 +1,124 @@
+# RFC-013: New initiative process
+
+
+## Proposed by
+
+Ryan Macklin ([@macklin](https://thegooddocs.slack.com/team/U01DYRWG43X))
+
+Initially submitted on 24 Aug 2022
+
+## Current status
+
+- [x] Draft
+- [ ] Under discussion
+- [ ] Final comment and voting (until YYYY-MM-DD) {{Add date after selecting this status.}}
+- [ ] Accepted
+- [ ] Rejected
+- [ ] Implemented
+- [ ] Deferred
+- [ ] Withdrawn
+
+
+## Proposal overview
+
+The Good Docs Project has an issue of delivery: we haven't delivered a 1.0 of our templates since the inception. And as a volunteer org, we need to keep focus on our mission and goal. It's easy to dilute ourselves if we don't keep ourselves accountable to focus and intentional communal growth.
+
```

Comment:
This is not directly related to the overall content of the RFC, so please treat this is a non-blocking side comment. I wanted to say that I don't agree with the narrative/framing here and so I want to push back a little. This paragraph makes it sound like we haven't accomplished anything noteworthy in our community and I find myself bristling a bit at that statement. While it is true that we haven't delivered a 1.0 release, we are quite close to achieving it.

I also want to state that we have many significant accomplishments that shouldn't be lightly overlooked. Since the project began in 2019, we've been laying important groundwork to attract high quality community members to TGDP because we've been building a healthy, vibrant, fun community that delivers very real tangible benefits to people who participate over the long term in both intrinsic and extrinsic terms. Because of our strong mentorship program, we're training the next generation of tech workers and have at least 5+ success stories of people who have gotten jobs or entered the field of tech because of their direct involvement in the project. We're also promoting thought leadership within the documentation field in general. I'm proud of the community we've built and I think that is still a significant achievement. Healthy and happy open source communities are hard to build and maintain, but that's exactly what we've done here.

To me, TGDP feels like the equivalent of a startup that has taken time to set up the business infrastructure and attract a high quality workforce. We've moved past the bootstrapping phase where we operate out of someone's garage into a fairly serious project. The difference is just that time passes more slowly in open source than it does in the business world and we have to module our expectations for how fast things get done with a volunteer community. 

I also think we have been productive, it's just not immediately visible what has been achieved. For the past two years, we've been laying the groundwork for how we will work, building out our tech stack, procedures, and communication channels so that we can ensure easier onboarding, good community support and feedback, and high quality products. I think we are only just now beginning to see the benefits of that foundational groundwork we've been building. If I may be so bold, I think we're 75% on target to having what would meet my definition of a 1.0 release and that is no small accomplishment.

At the end of the day, we've done great things and soon we will have something tangible to show for it. We're so, so close. Let's both celebrate what we've accomplished in the past while also setting our sights on the next milestone.

Reply:
*stealing this for a future blog post about the project :ninja:

Reply:
That's all pretty fair! I'll reframe the RFC with those thoughts in mind, and make sure we are being celebratory.

Reply:
And while I still agree with my "issue of delivery" sentiment, I also recognize (and should be explicitly saying) that we are working to deliver. We aren't just sitting around talking about a thing we'd like to do—we're doing the work. Just gotta make sure we keep doing the right work at the right time.

Timing is 



everything

Reply:
Adding to @barbaricyawps 's list of achievements. We actually have a number of templates which are at 1.0 quality, or about to be released at 1.0 quality. (We just don't have a full set yet.)

Reply:
Revised this section!